### PR TITLE
Symfony Web App Update 01-login.md

### PR DESCRIPTION
**Pull request by alexisluque**

### CODE REVIEW

**Comment by aaguiarz** [comment]
@alexisluque can you please resolve conflicts?

File: articles/quickstart/webapp/symfony/01-login.md [hunk 1]
```diff
@@ -193,18 +186,59 @@ Notice that we need to identify the user provided selected in the step before bo
...
+    {
+        $returnTo = sprintf('%s://%s:%s/auth0/logout',
+            $this->container->get('router')->getContext()->getScheme(),
+            $this->container->get('router')->getContext()->getHost(),
+            $this->container->get('router')->getContext()->getHttpPort());
+        $logoutUrl = sprintf(
+            'https://%s/v2/logout?client_id=%s&returnTo=%s',
+            '${account.namespace}',
+            '${account.clientId}',
+            $returnTo);
```
Comment (joshcanhelp) [inline articles/quickstart/webapp/symfony/01-login.md]: Ending paren on new line here

File: articles/quickstart/webapp/symfony/01-login.md [hunk 2]
```diff
@@ -193,18 +186,59 @@ Notice that we need to identify the user provided selected in the step before bo
...
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    /**
+     * @Route("/", name="homepage")
+     */
+    public function indexAction()
+    {
+        $returnTo = sprintf('%s://%s:%s/auth0/logout',
```
Comment (joshcanhelp) [inline articles/quickstart/webapp/symfony/01-login.md]: We probably don't want to include the colon here if there is no port number (which is the majority of cases that aren't localhost). I'm just worried that someone will copy this into their code and wonder why it's not working.

File: articles/quickstart/webapp/symfony/01-login.md [hunk 3]
```diff
@@ -193,18 +186,59 @@ Notice that we need to identify the user provided selected in the step before bo
...
     </a>
 {% else %}
     <h1>Symfony Auth0 Quickstart</h1>
     <a href="/connect/auth0"><button>Login</button></a>
 {% endif %}
 ```
+
+## Logout
+
+To log the user out, you have to clear the data from the session, and redirect the user to the Auth0 logout endpoint. You can find more information about this in the [logout documentation](/logout).
```
Comment (joshcanhelp) [inline articles/quickstart/webapp/symfony/01-login.md]: No comma before "and"

File: articles/quickstart/webapp/symfony/01-login.md (hunk 2 above)
Comment (joshcanhelp) [reply articles/quickstart/webapp/symfony/01-login.md]: Also ... I assume that `/auth0/logout` is where the application logout happens, right? That path is set somewhere else in the seed project?

File: articles/quickstart/webapp/symfony/01-login.md [hunk 4]
```diff
@@ -193,18 +186,59 @@ Notice that we need to identify the user provided selected in the step before bo
...
+            $returnTo);
+        return $this->render('default/index.html.twig', array(
+            'logoutUrl' => $logoutUrl
+        ));
+    }
+}
+```
+
+::: note
+Please take into consideration that the return to URL needs to be in the list of Allowed Logout URLs in the settings section of the application as explained in [our documentation](/logout#redirect-users-after-logout)
```
Comment (joshcanhelp) [inline articles/quickstart/webapp/symfony/01-login.md]: "Please take into consideration that the return to URL needs to be in the list of Allowed Logout URLs" -> "The return to URL must be in the list of Allowed Logout URLs..."

**Review by joshcanhelp — CHANGES_REQUESTED** [review]
Minor changes ... thank you!

File: articles/quickstart/webapp/symfony/01-login.md (hunk 2 above)
Comment (alexisluque) [reply articles/quickstart/webapp/symfony/01-login.md]: You're right, `/auth0/logout` is where the application logout happens. That path is set in `routing.yml` and `security.yml`.

File: articles/quickstart/webapp/symfony/01-login.md [hunk 5]
```diff
@@ -193,18 +186,62 @@ Notice that we need to identify the user provided selected in the step before bo
...
+
+class DefaultController extends Controller
+{
+    /**
+     * @Route("/", name="homepage")
+     */
+    public function indexAction()
+    {
+        $port = $this->container->get('router')->getContext()->getHttpPort();
+        $port == "" ? : $port = ':'.$port;
```
Comment (joshcanhelp) [inline articles/quickstart/webapp/symfony/01-login.md]: Confusing ... consider:

```php
if (!empty($port)) {
  $port = ':'.$port
}
```

**Review by joshcanhelp — APPROVED** [review]

**Pull request closed without merging**